[31m[1mERROR   [0m root:pre_sample_test.py:60 None